### ISSUE TITLE
[docs] a symbol cannot be an unstropped keyword

### ISSUE THREAD

**Issue by AmjadHD** [nim-lang/Nim#15963]
![image](https://user-images.githubusercontent.com/33235928/99129290-6a9c7180-260d-11eb-8e87-28764a1118b2.png)
![image](https://user-images.githubusercontent.com/33235928/99129296-6d976200-260d-11eb-97c9-a77f3e513332.png)
![image](https://user-images.githubusercontent.com/33235928/99129301-72f4ac80-260d-11eb-9362-7fa9429c9ca7.png)

**Comment by timotheecour** [comment]
use text, not images (impossible to search images on github)

**Comment by Araq** [comment]
Bug in the spec.

**Comment by AmjadHD** [comment]
Just a note `KEYW` is not explicitly defined in the manual, at least add a note that `KEYW` denotes the list of reserved keywords. It's not clear whether it includes operators like `div` for example.

**Comment by metagn** [comment]
Related, but probably doesn't need it's own issue

```
breakStmt = 'break' optInd expr?
continueStmt = 'break' optInd expr?
```

Second one should be 'continue'. `expr?` is correct.

**Comment by AmjadHD** [comment]
does `COMMENT` mean only block comments ?

```
colcom = ':' COMMENT?
blockExpr = 'block' symbol? colcom stmt
```
block comment:
```nim
let a = block: #[Works]# 1
```
line comment:
```nim
let a = block: # Indentation Error
1
```

**Comment by metagn** [comment]
This was fixed in #21655
